server/db/models/users.js

```diff
@@ -0,0 +1,258 @@
+//define Users model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
```

Comment:
the bracket notation is not needed here, I believe:

**const db  = require('./../database.js');**